### PR TITLE
Tab name changes and options moved

### PR DESCRIPTION
**Pull request by teryhill**
Tab name changes and options moved #347 #125 

I added a tab called lab, it is blank and we can remove it if it is not wanted

### CODE REVIEW

**Comment by tmccormi** [comment]
You have conflicts now that I have merged my default tabs changes ... please rebase...

**Comment by teryhill** [comment]
done

File: library/globals.inc.php [hunk 1]
```diff
@@ -1263,6 +1288,14 @@
        'Lab Report',                    // default
       xl('Document category name for storage of electronically received MDM documents.')
     ),
+
+    'hide_document_encryption' => array(
```
Comment (tmccormi) [inline library/globals.inc.php]: why is this under "Lab Report"?

File: library/globals.inc.php [hunk 2]
```diff
@@ -2208,128 +1967,20 @@
...
-            2 => xl('Response Only'),
-            3 => xl('Request & Response'),
-        ),
-         '0',
-        xl('Log all NewCrop eRx Requests and / or Responses.'),
     ),
+  
+    // Mail Tab
+  //
+  'Mail' => array(
```
Comment (tmccormi) [inline library/globals.inc.php]: Mail and SMS?

File: library/globals.inc.php [hunk 3]
```diff
@@ -2518,10 +2304,125 @@
        '30',
       xl('Rx Bottom Margin (px)')
     ),
+    
+    'erx_enable' => array(
+      xl('Enable NewCrop eRx Service'),
+      'bool',
+      '0',
+      xl('Enable NewCrop eRx Service.') + ' ' +
+      xl('Contact Medical Information Integration, LLC at http://mi-squared.com or ZH Healthcare at http://zhservices.com for subscribing to the NewCrop eRx service.')
```
Comment (tmccormi) [inline library/globals.inc.php]: Should we drop ZH or have them contact the community for a referral?

File: library/globals.inc.php [hunk 4]
```diff
@@ -2820,46 +2721,180 @@
...
+    ),
+    
+    'print_command' => array(
+      xl('Print Command'),
+      'text',                           // data type
+      'lpr -P HPLaserjet6P -o cpi=10 -o lpi=6 -o page-left=72 -o page-top=72',
+      xl('Shell command for printing from the server.')
+    ),
+    
+    'enable_hylafax' => array(
```
Comment (tmccormi) [inline library/globals.inc.php]: I think hylafax and Scanner should be a separate tab

**Comment by tmccormi** [comment]
see comments

**Comment by teryhill** [comment]
On it

File: library/globals.inc.php (hunk 1 above)
Comment (teryhill) [inline library/globals.inc.php]: The better question is why aren't the lab ones under lab

Comment (tmccormi) [inline library/globals.inc.php]: sure

File: library/globals.inc.php (hunk 2 above)
Comment (teryhill) [inline library/globals.inc.php]: Or do you want a separate tab for SMS

File: library/globals.inc.php (hunk 3 above)
Comment (teryhill) [inline library/globals.inc.php]: how about we just say "Contact the community for information on subscribing to the NewCrop eRx service."

File: library/globals.inc.php (hunk 2 above)
Comment (tmccormi) [inline library/globals.inc.php]: I think Mail and SMS can be one tab.   Does the SMS stuff even work?  what's required to make it work?   It used to be MaviQ, but that is no longer functioning (I know 'cause it was an MI2 thing)

File: library/globals.inc.php (hunk 4 above)
Comment (teryhill) [inline library/globals.inc.php]: one tab for Fax and one tab for Scanner

File: library/globals.inc.php (hunk 3 above)
Comment (tmccormi) [inline library/globals.inc.php]: good by me

File: library/globals.inc.php (hunk 2 above)
Comment (teryhill) [inline library/globals.inc.php]: Then let's just leave it as mail and remove the SMS stuff

File: library/globals.inc.php (hunk 3 above)
Comment (teryhill) [inline library/globals.inc.php]: done

File: library/globals.inc.php (hunk 2 above)
Comment (teryhill) [inline library/globals.inc.php]: should we also remove the phone stuff?

Comment (tmccormi) [inline library/globals.inc.php]: Yes, that was maviq too...

**Comment by teryhill** [comment]
Should we split the calendar and flow board?

**Comment by tmccormi** [comment]
I think the calendar and flowboard are closely related OK to leave them together.

**Comment by aethelwulffe** [comment]
Yeah, just re-label it "Post-Nuke_Flowboard"

**Comment by aethelwulffe** [comment]
Thumbs and tums.  Looks good, and can still be changed later...

**Pull request merged**